common/shlibs

```diff
@@ -48,6 +48,7 @@ libpcprofile.so glibc-2.32_1
 libcidn.so.1 glibc-2.32_1
 libmvec.so.1 glibc-2.32_1
 libz.so.1 zlib-1.2.3_1
+libz.so.1 zlib-ng-2.0.1_1
```

Comment:
reading `common/shlibs` stops at the first match, so this entry shouldn't exist.

Reply:
@ericonr OK, but if I remove this line, then `xbps-src` will emit warning about SONAME or something.

Reply:
It's just a warning.